src/pug/header.pug

```diff
@@ -0,0 +1,126 @@
+              li
+                a.dropdown__link(href='tel: +380967777777') (095) 77 77 777
+              li
+                a.dropdown__link(href='tel: +380969999999') (096) 99 99 999
+    nav.header-nav
+      ul.plain-text.fw-700.header-nav__list
+        li.header-nav__item.d-none
+          a.header-nav__link(href='') Головна
+        li.header-nav__item.d-none
+          a.header-nav__link(href='') АкцІї
+        li.header-nav__item.d-none
+          a.header-nav__link(href='') Доставка та оплата
+        li.header-nav__item.d-none
+          a.header-nav__link(href='') Про нас
+        li.header-nav__item.d-none
+          a.header-nav__link(href='') Контакти
+  hr
+  .header-cap-lower
+    .container
+      .row.d-flex
+        .col-6
+          a.logo-link-min(href='')
+            img(src='./img/logo-min.svg' alt='')
+          a.logo-link(href='')
+            img(src='./img/logo.svg' alt='')
+        .col-6
+          .plain-text.fw-700.d-flex.justify-content-end
+            a.shopping-cart(href='')
+              span.icon-shopping-cart.shopping-cart__icon
+              span.shopping-cart__text Кошик
+            .header-button-wrap
+              button.main-button Замовити дзвінок
+      nav.header-nav.d-none
+        ul.plain-text.fw-700.header-nav__list
+          li.header-nav__item.pl-0
+            a.header-nav__link(href='') Дивани
+          li.header-nav__item
+            a.header-nav__link(href='') ліжка
+          li.header-nav__item
+            a.header-nav__link(href='') Крісла
+          li.header-nav__item
+            a.header-nav__link(href='') комоди
+          li.header-nav__item
+            a.header-nav__link(href='') шафи
+          li.header-nav__item
+            a.header-nav__link(href='') Кухня
+  .container.header-cap-lower--xxxl
+    .row.d-flex
+      .col-xl-4.col-xxxl-5
+        nav.header-nav.d-none
+          ul.plain-text.fw-700.header-nav__list
+            li.header-nav__item.pl-0
+              a.header-nav__link(href='')
+                | Дивани
+                span.icon-vector.vector-show.pl-5
+            li.header-nav__item
+              a.header-nav__link(href='')
+                | Ліжка
+                span.icon-vector.vector-show.pl-5
+            li.header-nav__item
+              a.header-nav__link(href='')
+                | Крісла
+                span.icon-vector.vector-show.pl-5
+            li.header-nav__item
+              a.header-nav__link(href='')
+                | Комоди
+                span.icon-vector.vector-show.pl-5
+            li.header-nav__item
+              a.header-nav__link(href='')
+                | Шафи
+                span.icon-vector.vector-show.pl-5
+            li.header-nav__item
+              a.header-nav__link(href='')
+                | Кухня
+                span.icon-vector.vector-show.pl-5
+      .col-xl-4.xxxl-col-3.d-flex.justify-center
+        a.logo-link(href='')
+          img.max-width-100(src='./img/logo.svg' alt='')
```

Comment:
you don't need `max-width-100`. instead of that add `max-width: 100%` property for all img, picture, video elements